content/fast/reduce-network-payloads-using-text-compression/codelab-text-compression-brotli.md

```diff
@@ -0,0 +1,243 @@
+---
+page_type: glitch
+title: Enable Brotli compression
+author: mdiblasio
+description: |
+  In this codelab, learn how Brotli compression can further reduce compression ratios and your app's overall size.
+web_updated_on: 2019-05-05
+web_published_on: 2019-05-05
+glitch: fav-kitties-compress-starter
+---
+
+This codelab is an extension of the [Minify and compress network payloads codelab](/fast/reduce-network-payloads-using-text-compression/codelab-text-compression) and assumes you are familiar with the basics concepts of compression. As compared to other compression algorithms like `gzip`, this codelab explores how Brotli compression can further reduce compression ratios and your app’s overall size.
```

Comment:
I concur with Houssein's suggestion that it might make sense to update the title of the other codelab to 'Minify and compress network payloads with gzip' and this one compliments it with 'Minify and compress network payloads with brotli'. A nice to have but not critical.